src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1

```diff
@@ -168,6 +168,32 @@ class AzSKADOServiceMapping: CommandBase
                             }
                         }
                     }
+                    if($serviceConnEndPointDetail -and ![Helpers]::CheckMember($serviceConnEndPointDetail, "serviceEndpointExecutionHistory"))
```

Comment:
we should check for those connection also in which execution history is found but service tree mapping is not  found in any build/release

Reply:
keep this newly added login inside try catch